app/scripts/tabs.ts

```diff
@@ -209,6 +213,14 @@ export class TabModule {
         this.shortcut.addEventListener("Meta+8", goToTab);
         this.shortcut.addEventListener("Meta+9", goToTab);
 
+        const goToPreviousTab = (e) => {
+            if(this._previousTab)
+                this.slext.selectFile(this._previousTab.file);
+            e.preventDefault();
+        };
+
+        this.shortcut.addEventListener("Meta+0", goToPreviousTab);
```

Comment:
The default (Alt) setting of the meta key + 0 clashes with the Ace Editor "Fold All" command. Maybe we can use another shortcut here instead. Meta+B (for back)?

Reply:
Sure I can change this

Reply:
Done